test-cases/performance/perf-regression-latency-lwt-big.yaml

```diff
@@ -0,0 +1,36 @@
+test_duration: 300
+prepare_write_cmd: [
+  "cassandra-stress user profile=/tmp/cs_lwt_perf_big.yaml 'ops(insert=1)' no-warmup cl=QUORUM n=60000000 -port jmx=6868 -mode cql3 native -rate threads=100"
```

Comment:
I checked with Glauber, Let's run multiple cassandra-stress instances with distinct -pop distribution to load more data:
* first instance does -pop seq=1...X
* second instance -pop seq=X..2X\
and so on.

Reply:
1. Don't use CL=ANY, but change to CL=ALL.
We need the test to be consistent, otherwise we may get read repairs while querying the data later which will affect performance.

2. I agree we need to load more data, however I'm not sure that splitting the range with -pop is working with user profile.
It works for default stress and we do it in other tests.